pybricks/pybricks.c

```diff
@@ -93,9 +89,30 @@ void pb_package_import_all(void) {
     #endif
 }
 
-void pb_package_pybricks_init(void) {
-    // Reset additions to Color parameters
-    pb_type_Color_reset();
+/**
+ * Prepares Pybricks MicroPython environment.
+ *
+ * @param [in]  import_all      Whether to import * from all pybricks.* modules.
+ * @return                      True on success, false if an exception was raised.
+ */
+
+bool pb_package_pybricks_init(bool import_all) {
```

Comment:
It looks like we are not checking the return value anywhere, so we should either check it or change it to `void`.